db/schema.rb

```diff
@@ -56,6 +57,7 @@
     t.string   "question"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer  "order"
```

Comment:
So this file should be reverted to the previous state or if you do something in a migration, commit this file when it's overwritten.